samples/apache-others/challenge/config/apache2-nsjail-others.conf

```diff
@@ -0,0 +1,9 @@
+ServerName kctf-nsjail-php
+Listen 1337
+User user
+
+<VirtualHost *:1337>
+ DocumentRoot "/var/www/html"
+ ProxyPreserveHost On
+ ProxyPass "/nodejs" "http://localhost:8081/"
```

Comment:
we have an apache server in front so that we can host multiple backends at different paths.

another way to set this up, however, could be to just have nsjail listen on port 1337.